lib/index.js

```diff
@@ -15,7 +15,10 @@ const {
   optionallyAuthenticated
 } = brPassport;
 const uuid = require('uuid/v4');
-require('bedrock-validation');
+const {validate} = require('bedrock-validation');
+// FIXME add an implementation in bedrock-express
```

Comment:
Are we fixing this?

Reply:
Not now, no.

Reply:
at the moment these are specific to just this api so probably not.
if another api needs boolParser or intParser we might consider it then.